CliWrap/StandardBufferHandler.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+
+namespace CliWrap
+{
+    /// <summary>
+    /// The standard implementation of <see cref="IStandardBufferHandler"/>
+    /// </summary>
+    public class StandardBufferHandler : IStandardBufferHandler
+    {
+        private readonly Action<string> _standardOutputHandler;
+        private readonly Action<string> _standardErrorHandler;
+
+        /// <summary>
+        /// Sets up a standard buffer handler using two callbacks. Both callbacks are optional.
+        /// </summary>
+        /// <param name="standardOutputHandler">The callback to use when it gets written to the standard output</param>
```

Comment:
`The callback to use when it gets written...`
maybe change to `The callback to use when a line is written...` to make it more clear